src/components/Home.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable indent */
+/* eslint-disable react/no-array-index-key */
+/* eslint-disable no-unused-vars */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import { useSelector, useDispatch } from 'react-redux';
+import React, { useEffect, useState } from 'react';
+import {} from '../store/rootReducer';
+import { Link } from 'react-router-dom';
+import fetchData from '../store/actions';
+
```

Comment:
- [ ] As requested by the previous reviewer, Instead of disabling eslint, please kindly fix the issue since the linter is there to help you implement best practices. please fix them.

Reply:
@Emmyn5600 It has been removed -- but I think it's not reflecting on GH just yet. Kindly approve the project.